athf/core/llm_provider.py

```
@@ -281,7 +281,7 @@ def _get_client(self) -> Any:
             import boto3
             from botocore.config import Config
         except ImportError:
-            raise ImportError("boto3 package is not installed. Install it with: pip install boto3")
+            raise ImportError("boto3 package is not installed. Install it with: pip install 'agentic-threat-hunting-framework[bedrock]'")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)